spec/grape/api_spec.rb

```diff
@@ -1289,7 +1289,7 @@ def three
       subject.get '/exception' do
         fail 'rain!'
       end
-      expect { get '/exception' }.to raise_error
+      expect { get '/exception' }.to raise_error(RuntimeError)
```

Comment:
Since we're at it, you should explicitly check that the error raised is the one we threw, so not just `RuntimeError`, but the same message.

```ruby
expect { get '/exception' }.to raise_error RuntimeError, 'rail!'
```

Adjust the others too please?